### PR TITLE
add Out of the box section to the About page

### LINKED ISSUE #1355: Extend the "About page" by OOTB-Section
**Description**

Add a section **2 - Out of the Box Features** to the [About Page](https://luigi-project.io/about) that should look&feel like the 02-Benefits section on the [main page](https://luigi-project.io)
For the new section, please use [this content](https://github.com/SAP/luigi/blob/0283479e37e985094c5531a009d7245a7805bd64/docs/luigi-features-overview.md).

### PR DESCRIPTION
**Pull request by UlianaMunich**
Fixes #1355 

Add new section "Out of the box features" tot he About page

### CODE REVIEW

File: website/landingpage/dev/src/partials/outoftheboxfeatures.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
+
+  <section class="s-benefits">
+    <div class="row">
+      <div class="column small-12 large-12">
```
Comment (marynaKhromova) [inline website/landingpage/dev/src/partials/outoftheboxfeatures.html:4]: `small-12` and `large-12` do the same. It's enough just to use `small-12`

File: website/landingpage/dev/src/partials/outoftheboxfeatures.html [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            <h3 class="title-3">Pre-set UI</h3>
+            <p class="description">
+                Do you want a standardized UI for your app which is easy to use and configure? Luigi comes with 
+                an out-of-the-box L-shaped shell bar with a variety of different menus, dropdowns and switchers. 
+                The navigation built with <a href="https://sap.github.io/fundamental-styles/">Fundamental Styles</a> allows all your micro frontends to be embedded in a 
+                consistent UI. Learn more about Luigi navigation <a href="https://docs.luigi-project.io/docs/navigation-configuration">here</a>.
+            </p>
+          </div>
+          <div class="column small-12 large-3 image">
+            <img src="{{root}}assets/img/presetui.png" alt="Luigi - Micro Frontend Framework - Pre-set UI" />
```
[the remark this answers is not in the document]
Comment (UlianaMunich) [reply website/landingpage/dev/src/partials/outoftheboxfeatures.html]: SVGs were not on a Sharedrive, it was requested yesterday and updated

File: website/landingpage/dev/src/assets/scss/components/_home_page.scss [hunk 3]
```diff
@@ -52,6 +52,13 @@
 
     .description {
       margin-left: 30px;
+      a {
+        font-weight: 600;
+        color: #3c4553;
```
Comment (marynaKhromova) [inline website/landingpage/dev/src/assets/scss/components/_home_page.scss:57]: Please use variables for colors. Take a look at `assets/scss/_settings.scss` file. There is a `$foundation-palette` variable.

File: website/landingpage/dev/src/partials/features.html [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+      <div class="column small-12 large-8 xlarge-6">
+        <div class="sub-title has-number">
+            <span class="text">Out of the Box Features</span>
+        </div>
+        <h2 class="title-2 has-big-margin">Make your app as consistent and easy to develop as possible</h2>
+      </div>
+    </div>
+  </section>
+
+  <section>
```
Comment (marynaKhromova) [inline website/landingpage/dev/src/partials/features.html:13]: You can simply remove `<section>` here.

File: website/landingpage/dev/src/partials/features.html [hunk 5]
```diff
@@ -0,0 +1,140 @@
+
+  <section class="is-centered">
```
Comment (marynaKhromova) [inline website/landingpage/dev/src/partials/features.html:2]: You can remove `<section>` here and move `is-centered` class to the `row`

File: website/landingpage/dev/src/assets/scss/components/_home_page.scss (hunk 3 above)
Comment (UlianaMunich) [reply website/landingpage/dev/src/assets/scss/components/_home_page.scss]: done

File: website/landingpage/dev/src/partials/features.html (hunk 4 above)
Comment (UlianaMunich) [reply website/landingpage/dev/src/partials/features.html]: done

File: website/landingpage/dev/src/partials/features.html (hunk 5 above)
Comment (UlianaMunich) [reply website/landingpage/dev/src/partials/features.html]: done

**Pull request merged**